3rdparty/clapack/blas/daxpy.c

```diff
@@ -0,0 +1,107 @@
+/* daxpy.f -- translated by f2c (version 20061008).
```

Comment:
If you have manually converted the Fortran code to C, there is a new [release 3.9.0](https://github.com/Reference-LAPACK/lapack/releases) that can be considered in the future.

If this is already the newest version, I think that the `LAPACK_MAJOR/MINOR/PATCH_VERSION` should be updated.

If it is indeed the 3.2.1 version, this version is a little bit too old (2009). Maybe in the future it could be interesting to see how to do the conversion.

Reply:
I used version 3.2.1 from [here](https://www.netlib.org/clapack/).
I didn't know that there exist 3.9.0 that I need definitively to consider.